docs/integrate/etl/index.md

```diff
@@ -72,6 +72,9 @@ Tutorials and resources about configuring the managed variants, Astro and CrateD
 
 - {ref}`integrate-mongodb`
 
+## InfluxDB
+
+- {ref}`integrate-influxdb`
```

Comment:
We are enumerating all items sorted alphanumerically ascending across the board. Thanks!